backend/main.py

```
@@ -45,10 +46,10 @@ async def start_background_tasks(self):
 
             await self.queue_manager.start(num_workers=3)
 
-            # --- Load commands inside the async startup function ---
+            # --- Load commands inside the async startup function --- 
             try:
                 await self.discord_bot.load_extension("integrations.discord.cogs")
-            except (ImportError, commands.ExtensionError) as e:
+            except (ImportError, discord.errors.ExtensionFailed) as e:
                 logger.error("Failed to load Discord cog extension: %s", e)
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)